src/poprox_recommender/components/embedders/article.py

```diff
@@ -20,6 +21,12 @@
 TITLE_LENGTH_LIMIT = 30
 
 
+@dataclass
+class NRMSArticleEmbedderConfig:
+    model_path: PathLike
+    device: str | None
```

Comment:
This (and the user config) are dataclasses, instead of Pydantic models, because names starting with `model_` are reserved in Pydantic models, but are not reserved in the Pydantic dataclass machinery. Pydantic and LensKit support both `BaseModel` and dataclass configuration classes.

We may want to use dataclass for all component configurations for consistency.